toolchain/check/testdata/class/syntactic_merge_literal.carbon

```diff
@@ -17,6 +15,8 @@ class D(b:! C(1_000));
 class D(b:! C(1_000)) {}
 
 // --- fail_int_mismatch.carbon
+// CHECK:STDERR: fail_int_mismatch.carbon: error: `Main//default` previously provided by `int_match.carbon` [DuplicateMainApi]
```

Comment:
Looks like this file needs another round of autoupdate.

Separate from this PR, I wonder if we should have some kind of check to prevent `DuplicateMainApi` appearing in our test files?

Reply:
I'm mixed on that -- maybe if it's better placed we'll just catch it when needed. As is we'd need to either delete tests or have it be slightly complex, with special-casing for the files which are deliberately testing DuplicateMainApi

Reply:
Fixing this case in #4678 (didn't feel okay doing it here, since you'd enabled auto-merge)